dist/avatar/avatar.css

```diff
@@ -1,17 +1,34 @@
 .avatar {
     align-items: center;
-    background-color: #84b4fb;
     border-radius: 50%;
-    color: #002a69;
+    display: flex;
     display: inline-flex;
```

Comment:
Duplicate `display`.

Reply:
Done! Was due to mixin also having a default display.